account_financial_report_qweb/__openerp__.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Author: Julien Coux
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'QWeb Financial Reports',
+    'version': '9.0.1.0.0',
+    'category': 'Reporting',
+    'summary': 'OCA Financial Reports',
+    'author': 'Camptocamp SA,'
+              'initOS GmbH,'
+              'redCOR AG,'
+              'Odoo Community Association (OCA)',
+    "website": "https://odoo-community.org/",
+    'depends': [
+        'account',
```

Comment:
@ovnicraft that's arguable, since some menus from `account` are modified in this module ; IMO, it's clearer to have `account` as a "direct" dependency, even if technically it is not needed.